src/platform/Ameba/FactoryDataProvider.cpp

```diff
@@ -664,5 +665,19 @@ CHIP_ERROR FactoryDataProvider::GetRotatingDeviceIdUniqueId(MutableByteSpan & un
     return err;
 }
 
+CHIP_ERROR FactoryDataProvider::GetProductFinish(ProductFinishEnum * finish)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    *finish        = ProductFinishEnum::kOther;
+    return err;
+}
+
+CHIP_ERROR FactoryDataProvider::GetProductPrimaryColor(ColorEnum * primaryColor)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    *primaryColor  = ColorEnum::kBlack;
```

Comment:
@pankore Is there a reason this is not just returning error?  Color is optional to return....